tasks/pipeline-pip-deps.yml

```diff
@@ -9,12 +9,13 @@
     - "archivematica-mcp-client"
     - "archivematica-mcp-server"
 
-- name: "virtualenv | Create environment and install requirements"
+- name: "Create virtualenv and install pip-tools"
   pip:
     virtualenv: "{{ archivematica_src_am_virtualenv }}"
-    requirements: "{{ archivematica_src_dir }}/archivematica/{{ 'requirements-dev.txt' if is_dev else 'requirements.txt' }}"
+    name: "pip-tools"
```

Comment:
Should we also pin `pip-tools` version to avoid potential [compatibility version issues](https://github.com/jazzband/pip-tools/#versions-and-compatibility) with `pip`?

Reply:
Good idea. I've made the change and tested it.